test/parallel/test-accessor-properties.js

```diff
@@ -36,6 +24,11 @@ const UDP = internalBinding('udp_wrap').UDP;
   const properties = ['bytesRead', 'fd', '_externalStream'];
 
   properties.forEach((property) => {
+    // Should throw instead of raise assertions
+    assert.throws(() => {
+      TTY.prototype[property];
+    }, TypeError, `TypeError on: TTY.prototype.${property}`);
```

Comment:
This change means we need to add a custom message or else we won't know which property is the issue. To me, this is another reason not to do it. But if we're going to do it:

```suggestion
    }, TypeError, `Missing expected TypeError for TTY.prototype.${property}`);
```